### PR TITLE
Add Font tests

### PR DESCRIPTION
**Pull request by hughbe**

### CODE REVIEW

**Comment by hughbe** [comment]
Test Linux x64 Release Build
Test NETFX x86 Release Build

**Comment by hughbe** [comment]
Test Linux x64 Release Build

File: src/System.Drawing.Common/tests/FontTests.cs [hunk 1]
```diff
@@ -19,5 +301,575 @@ public void Ctor_NoSuchFamilyName_SetsFamilyToGenericSansSerif(string familyName
...
+            {
+                Graphics graphics = Graphics.FromImage(image);
+                graphics.Dispose();
+
+                AssertExtensions.Throws<ArgumentException>(null, () => font.ToLogFont(new LOGFONT(), graphics));
+            }
+        }
+
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
+        public class LOGFONT
```
Comment (mellinoe) [inline src/System.Drawing.Common/tests/FontTests.cs]: I have a feeling this is going to be problematic when we start running against libgdiplus, but we can deal with that when it occurs.

**Comment by mellinoe** [comment]
LGTM. Thanks again for the tests and cleanup!

**Pull request merged**